Lib/test/libregrtest/refleak.py

```diff
@@ -233,6 +235,11 @@ def dash_R_cleanup(fs, ps, pic, zdc, abcs):
     copyreg.dispatch_table.update(ps)
     sys.path_importer_cache.clear()
     sys.path_importer_cache.update(pic)
+    lcache, linteractive = linecache_data
```

Comment:
Can you add a code comment as to _why_ this is needed?

Reply:
👍